pkg/connector/push.go

```diff
@@ -105,13 +105,18 @@ type connectBackgroundEvent struct {
 }
 
 func (m *MetaClient) ConnectBackground(ctx context.Context, params *bridgev2.ConnectBackgroundParams) error {
+	l := zerolog.Ctx(ctx)
```

Comment:
Usually the variable is called `log`

Reply:
https://github.com/mautrix/meta/blob/d97005b8ab83228efe1f0a5e78af244d0ae6796f/pkg/messagix/events.go#L10 for example

Makes sense to me!